functions/src/lib/slack.ts

```diff
@@ -0,0 +1,144 @@
+import {WebClient} from '@slack/web-api';
+import config from './config';
+import {RequestAction} from './constants/requestAction';
+
+const {SLACK_OAUTH_TOKEN, SLACK_BOT_NAME, SLACK_PUBLIC_HOST_REQUESTS_CHANNEL} =
+  config;
+
+const createSlackClient = () => {
+  if (
+    SLACK_BOT_NAME &&
+    SLACK_PUBLIC_HOST_REQUESTS_CHANNEL &&
```

Comment:
I was more thinking that we should use `SLACK_PUBLIC_HOST_REQUESTS_CHANNEL` to determine if we should send the message. That way we can have some messages being sent, and some not. Checking for `SLACK_BOT_NAME`, `SLACK_OAUTH_TOKEN` here makes total sense thought!